models/users.js

```diff
@@ -54,11 +54,21 @@ const addOrUpdate = async (userData, userId = null) => {
  * @return {Promise<userModel|Array>}
  */
 const fetchUsers = async (query) => {
+  const allowedUserTypes = ['new', 'blocked', 'member']
+  let snapshot
   try {
-    const snapshot = await userModel
-      .limit(parseInt(query.size) || 100)
-      .offset((parseInt(query.size) || 100) * (parseInt(query.page) || 0))
-      .get()
+    if (typeof query.userType !== 'undefined' && allowedUserTypes.includes(query.userType)) {
```

Comment:
is it needed to check for `undefined` ?
cant we use something like 
`if(query.userType && ....)`